SimplenoteUITests/Generic.swift

```diff
@@ -63,6 +63,11 @@ func populateNoteList(title: String, numberToCreate: Int) {
     }
 }
 
+func getRandonId() -> String.SubSequence {
+    let id = NSDate().timeIntervalSince1970.description.suffix(6)
+    return id
+}
```

Comment:
Two points here:
1. A typo in function name
2. Is there a reason to have the return value as `String.Subsequence` and not `String`, taking into account that the return value is used as a string? (please let me know if there's a context I might be missing here)

```suggestion
func getRandomId() -> String {
    let id = NSDate().timeIntervalSince1970.description.suffix(6)
    return String(id)
}
```

Reply:
Yeah, returning String was my first attempt but I've got the `Cannot convert return expression of type 'String.SubSequence' (aka 'Substring') to return type 'String'` message from Xcode and just set the return type to `String.SubSequence` instead of just converting variable. Returning `String` is definitely better.